### PR TITLE
updating readme

### PR DESCRIPTION
**Pull request by mike-hoang**
## What does this PR do?

Move information from go/README.md to ./README.md since the JS and Java implementations are removed. 

### Which issue(s) does this PR fix

fixes https://github.com/redhat-developer/alizer/issues/215

### PR acceptance criteria

Testing and documentation do not need to be complete in order for this PR to be approved. We just need to ensure tracking issues are opened.

- [ ] Unit/Functional tests

  

- [x] Documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +16,114 @@ detect components (the concept of component is taken from Odo and its definition
...
+### CLI Arguments
+
+#### --log
+
+Sets the logging level of the CLI. The arg accepts only 3 values [`debug`, `info`, `warning`]. The default value is `warning` and the logging level is `ErrorLevel`. Example usage
+
+```shell
+$ ./alizer analyze --log debug <path>
+```
+
```
Comment (thepetk) [inline README.md:43]: * for devfile command we have the `--registry -r` arg which specifies the registry we will use.
* for the component command we have  `port-detection -p` ("--port detection strategy to use when detecting a port. Currently supported strategies are 'docker', 'compose' and 'source'. You can pass more strategies at the same time. They will be executed in order. By default Alizer will execute docker, compose and source") as it mentioned in the code.

File: README.md [hunk 2]
```diff
@@ -21,7 +16,114 @@ detect components (the concept of component is taken from Odo and its definition
...
+#### Component Detection
+
+It detects all components which are found in the source tree.
+
+```go
+import "github.com/redhat-developer/alizer/pkg/apis/recognizer"
+
+components, err := recognizer.DetectComponents("./")
```
Comment (thepetk) [inline README.md:63]: Having a consistent "your-projects-path" in all cases maybe could better?

File: README.md [hunk 3]
```diff
@@ -21,7 +16,114 @@ detect components (the concept of component is taken from Odo and its definition
...
+```
+
+#### Devfile Detection
+
+It selects a devfile from a list of devfiles (from a devfile registry or other storage) based on the information found in the source tree.
+
+```go
+import "github.com/redhat-developer/alizer/pkg/apis/recognizer"
+
+devfile, err := recognizer.SelectDevFileFromTypes("myproject", devfiles)
```
Comment (thepetk) [inline README.md:73]: Having a consistent "your-projects-path" in all cases maybe could better?

File: README.md [hunk 4]
```diff
@@ -21,7 +16,114 @@ detect components (the concept of component is taken from Odo and its definition
...
+### Library Package
+
+#### Language Detection
+
+To analyze your source code with Alizer, just import it and use the recognizer:
+
+```go
+import "github.com/redhat-developer/alizer/pkg/apis/recognizer"
+
+languages, err := recognizer.Analyze("./")
```
Comment (thepetk) [inline README.md:53]: Having a consistent "your-projects-path" in all cases maybe could better?

File: README.md [hunk 5]
```diff
@@ -21,7 +16,114 @@ detect components (the concept of component is taken from Odo and its definition
...
+
+#### --log
+
+Sets the logging level of the CLI. The arg accepts only 3 values [`debug`, `info`, `warning`]. The default value is `warning` and the logging level is `ErrorLevel`. Example usage
+
+```shell
+$ ./alizer analyze --log debug <path>
+```
+
+### Library Package
```
Comment (thepetk) [inline README.md:44]: Maybe a link here to name_detection.md , port_detection.md (or in the component detection section)?

**Review by thepetk — CHANGES_REQUESTED** [review]
I've left some minor comments. I really liked we renamed the `docs/go` to `docs/public` :)

File: README.md (hunk 5 above)
Comment (mike-hoang) [reply README.md:54]: added to the component detection section

**Review by thepetk — APPROVED** [review]

**Pull request merged**